bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java

```diff
@@ -1773,7 +1787,7 @@ private boolean supportsOnvifEvents() {
     private void keepMjpegRunning() {
         CameraServlet localServlet = servlet;
         if (localServlet != null && !localServlet.openStreams.isEmpty()) {
-            if (!mjpegUri.isEmpty() && !"ffmpeg".equals(mjpegUri)) {
+            if (!mjpegUri.isEmpty() && !"ffmpeg".equals(mjpegUri) && !mjpegUri.toLowerCase().startsWith("rtsp://")) {
```

Comment:
I believe you can use `!usingRtspForMjpeg()`

Reply:
Thanks that is easier to read.